teacher.rb

```diff
@@ -3,8 +3,8 @@
 class Teacher < Person
   attr_reader :specialization
 
-  def initialize(id, age, specialization, name = 'Unknown', parent_permission: true)
-    super(id, age, name, parent_permission)
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] There is an error while trying to create a teacher due to wrong number of arguments passed, kindly fix this
![image](https://user-images.githubusercontent.com/66097371/193245024-fe087711-b6f5-4316-a6b1-459af24fc0fa.png)

Reply:
Thank you, this has been fixed now